tests/Utilities.Test.NUnitTests/NUnitTestExamples.cs

```diff
@@ -0,0 +1,55 @@
+using NUnit.Framework;
+using Moq;
+
+#if NET5_0 || NETCOREAPP3_1 || NETFRAMEWORK
+using System;
+#endif
+
+namespace Matr.Utilities.Test.UnitTests
+{
+    public class NUnitTestExamples : TestBase
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void SimpleExample_WithDependency_ResolvesMockedDependency()
+        {
+            var random = new Random();
+            var returnValue = random.Next(1, int.MaxValue);
```

Comment:
Is leuk maar waarom Random? Voegt toch niks echt toe?

Reply:
Will always generate different but valid cases.
Though this is more of a demo project so agreed, does not have to be fancy.

Reply:
replaced with ```returnValue = 1```